cocoa/tests_backend/app.py

```diff
@@ -190,6 +190,14 @@ def activate_menu_close_all_windows(self):
     def activate_menu_minimize(self):
         self._activate_menu_item(["Window", "Minimize"])
 
+    def assert_dialog_in_focus(self, dialog):
+        # Cannot directly compare `dialog._impl.native`(`NSAlert`) and
+        # `self.app._impl.native.keyWindow`(`_NSAlertPanel`) objects.
+        # Hence, do a string comparison instead.
+        assert str(dialog._impl.native.objc_class.__name__) in str(
+            self.app._impl.native.keyWindow.objc_class.__name__
+        ), "The dialog is not in focus"
```

Comment:
No issue with a string comparison if it's needed... but doesn't `NSAlert.window` give you the window for the alert (which should be the keyWindow when the dialog is showing, AFAICT)?

Reply:
You are right. Thank you :)